app/com/gu/floodgate/contentsource/ContentSourceService.scala

```diff
@@ -1,12 +1,15 @@
 package com.gu.floodgate.contentsource
 
+import com.gu.floodgate.DynamoDBTable
+
 import scala.concurrent.Future
 
-class ContentSourceService {
+class ContentSourceService(contentSourceTable: DynamoDBTable[ContentSource]) {
```

Comment:
Not sure of the value of this service that just delegates to Dynamo. Unless you're planning to add some orchestration in here, you could get rid of it and just let the controller talk directly to the Dynamo class.